From 0cf9a4da866e52ae90e20266811cf48e7a3806e6 Mon Sep 17 00:00:00 2001
From: wjyrich <1071633242@qq.com>
Date: Sat, 7 Sep 2024 18:06:36 +0800
Subject: [PATCH] install

---
 variables.pri | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/variables.pri b/variables.pri
index 93e6328..6555313 100644
--- a/variables.pri
+++ b/variables.pri
@@ -77,7 +77,7 @@ win32:QMAKE_LFLAGS += -Wl,--enable-auto-import
 # Install root
 win32:INSTALLROOT       = $$(SystemDrive)/qlcplus
 macx:INSTALLROOT        = ~/QLC+.app/Contents
-unix:!macx:INSTALLROOT += /usr
+unix:!macx:INSTALLROOT += $$PREFIX
 android:INSTALLROOT     = /
 ios:INSTALLROOT         = /
 
@@ -252,7 +252,7 @@ android:USERCOLORFILTERSDIR    = $$USERDATADIR/colorfilters
 ios:USERCOLORFILTERSDIR        = $$USERDATADIR/ColorFilters
 
 # udev rules
-unix:!macx:UDEVRULESDIR = /etc/udev/rules.d
+unix:!macx:UDEVRULESDIR = $$PREFIX/etc/udev/rules.d
 
 # AppStream metadata
 unix:!macx:METAINFODIR = $$INSTALLROOT/share/metainfo
-- 
2.43.4

